tests/test_datadog.py

```diff
@@ -42,3 +44,55 @@ def test_api_key():
     configured_api_key = DataDog.settings[DataDog.KEY_DATADOG_API_KEY]
 
     assert configured_api_key == 'test_api_key'
+
+
+def test_metrics():
+    metric_prefix = test_metrics.__name__
+    DataDog.configure_settings({'DATADOG_STATS_ENABLED': True,
+                                'DATADOG_STATS_PREFIX': metric_prefix})
+
+    mock_dd = DataDog.stats()
+    for method in ['gauge', 'increment', 'decrement', 'histogram', 'event']:
+        setattr(mock_dd, method, mock.Mock())
```

Comment:
As far as I understand mock, it'll automatically return a new mock object when you call a method/attribute on a Mock instance. Since these are singletons, you should be able to have access to the mock that was called. But again, I might be overlooking something here. It's just a suggestion to simplify the test...which can safely be ignored :wink: